src/Microsoft.OpenApi/Any/OpenApiArray.cs

```diff
@@ -14,5 +15,22 @@ public class OpenApiArray : List<IOpenApiAny>, IOpenApiAny
         /// The type of <see cref="IOpenApiAny"/>
         /// </summary>
         public AnyType AnyType { get; } = AnyType.Array;
+
+        /// <summary>
+        /// Write out contents of OpenApiArray to passed writer
+        /// </summary>
+        /// <param name="writer">Instance of JSON or YAML writer.</param>
+        public void Write(IOpenApiWriter writer)
```

Comment:
public? not internal?

Reply:
Yes it needs to be public because it is part of an interface that will be implemented by any custom extension implementation.

---
In reply to: [161353844](https://github.com/Microsoft/OpenAPI.NET/pull/176#discussion_r161353844) [](ancestors = 161353844)